src/components/AgentAuthority/ContentWrap/ContentWrap.js

```diff
@@ -6,9 +6,12 @@
 import ColumnLeftWrap from '../ColumnLeft/ColumnLeftWrap'
 import ColumnRightWrap from '../ColumnRight/ColumnRightWrap'
 
+import IssueLicense from '../IssueLicense'
+
 export default function ContentWrap({ origin }) {
   return (
     <>
+      <IssueLicense origin={origin} />
```

Comment:
since we are using the left/right columns approach maybe this should be on either or rename columns to panel and middle have as a context? just a thought

Reply:
sorry @n3op2 not sure I understand this one?

Reply:
we wrap all the content in column right or left, however we have added this before other wrappers, so thought maybe it would make sense to keep in one of the columns of create another wrapper either instead or besides

Reply:
got it. I think the issue is it won't fit neatly into either and the reality is this should be on a backend. I'm happy leaving it like this as all it does is render an error (if there is one) and we'll remove it when this moves to the backend